presto-native-execution/README.md

```diff
@@ -123,6 +123,8 @@ Run CLion:
 * Edit menu CLion->Preferences->Build, Execution, Deployment->CMake
   * CMake options: `-DVELOX_BUILD_TESTING=ON -DCMAKE_BUILD_TYPE=Debug`
   * Build options: `-- -j 12`
+* If you want to enable Parquet and S3 support add these as well in CMake options,
```

Comment:
Should we make this a sub-item of the previous item above `* Build options: ....` ?
```   
* Optional CMake options to enable Parquet and S3 support: ` -DPRESTO_ENABLE_PARQUET=ON -DPRESTO_ENABLE_S3=ON`
```